raiden/tasks.py

```diff
@@ -179,4 +179,8 @@ def run_callbacks(self, current_block, chain_id):
         self.last_block_number = current_block
 
     def stop_async(self):
-        self.stop_event.set(True)
+        self._stop_event.set(True)
+
+    def stop(self, timeout=None):
```

Comment:
please don't add timeouts for the inner layers, only use it at the outer-most layers of the code

Reply:
It's just to comply with the interface. Inner layers may leave then as `None` as well